### ISSUE TITLE
Not all the data it seem to be

### ISSUE THREAD

**Issue by marcel-1984** [LordMike/MBW.BlueRiiot2MQTT#71]
Best Mike. 

Can you please help me? Ik have add and install the addon and I have some sensors;

![C0A5ABC8-EBAE-4FE8-AF4C-396BB354C9BA](https://user-images.githubusercontent.com/83901058/177408932-3b1a8fd0-a7f0-4884-80dd-b2d0018daffa.png)

In my MQTT the other sensors are Coming in;

![1F52F4A2-5F54-46CC-9249-20B4F0E943B4](https://user-images.githubusercontent.com/83901058/174448419-4435a4ed-828f-4ddf-9202-6cf276344dad.jpeg)

Only the vallue’s of that sensors ( ph, temp, orb ) are not visible in my dashboard. 

I hope you can help me with this. Of you need somthing, let me no.

Best regeards,
Marcel

**Comment by robertjan** [comment]
same issue on my side... hope Mike can help us out

**Comment by LordMike** [comment]
Hmm, that is odd. Is your BR sensor active and working, and otherwise reporting in values?

If it is, then it _should_ appear. You can try [raising the log levels](https://github.com/LordMike/MBW.BlueRiiot2MQTT#troubleshooting) and see what comes of it. I'd be interested in some of the json coming back from BlueRiiot, which should show us what's up.

**Comment by robertjan** [comment]
The blueriiot app on my iphone is working fine, i have premium subscription but no wifi externder

in the broker logfiles i see some strange behavior: 
![broker_logbook](https://user-images.githubusercontent.com/4417551/185360887-0aee2bb1-b635-4ebe-bd12-858ec2b44365.jpg)

Sorry but i`m not that familiar with HA yet, just a beginner so i`m not quite sure how the rais op the log levers.. sorry

this is a view within the MQTT explorer:
![MQTTexplorer](https://user-images.githubusercontent.com/4417551/185361999-044322b4-12d4-408c-980f-6a6add69de70.jpg)

**Comment by LordMike** [comment]
That is weird behavior. Do you know if the ip (..32.2) belongs to blue riiot ?

It could also be HA itself - or maybe you have a third application using mqtt.
________________________________

**Comment by LordMike** [comment]
Just saw the last picture. So only two clients, this and HA. 

That’s so weird. In the discovery document you’ve found, the state topic is set to “sarah/…” - but I have no idea why it would be that. It should be “blueriiot” by default. 

Could you post your config for blueriiot2mqtt?

**Comment by marcel-1984** [comment]
By me it still don’t work. 

![563A49B8-2982-4DD8-805D-A1EF3242B9B7](https://user-images.githubusercontent.com/83901058/185764480-50a18ed1-64da-44b7-ad0d-208e73c8608e.png)
![D8198068-9422-4A9D-93D4-A35EF2253FC4](https://user-images.githubusercontent.com/83901058/185764481-05764216-b2f9-4e69-82f3-27d198993b1c.png)
![3DB4F4B0-36C0-4B80-AD6F-E6CDB4960E4E](https://user-images.githubusercontent.com/83901058/185764482-a385cf27-7a94-4921-85f7-7c97e6ee7f22.png)
![41DD1C6F-8A7D-40B2-BB6E-873FF4CD404E](https://user-images.githubusercontent.com/83901058/185764484-e7539932-3019-4be0-a29e-9403595c3858.png)
![407E595B-C90E-4E12-B4BE-E1193ACE9340](https://user-images.githubusercontent.com/83901058/185764866-e64c315e-d9e9-44ed-8d4c-d9590e0b85a4.jpeg)

**Comment by robertjan** [comment]
I have quite the same setup as Marcel in MQTT, for the test i did change the sarah/ to blueriiot but als that did not change anything. 

Status is online but the sensors getting 0 on request

![sensor-error-message](https://user-images.githubusercontent.com/4417551/185798121-0632410a-c397-4693-8ee0-fff836f38bad.png)

**Comment by LordMike** [comment]
Hey again,

Sorry for the delay - I'm busy elsewhere. You didn't get this solved, did you?
I've booted up my Blue Riiot (my pool wasn't used this year), and it updates data just fine - so that parts working.

What's odd on your end is this "sarah" in your state_topic. For some reason, the discovery document (the one you showed) says that the state must be fetched from this topic (`sarah/`) - and in Robertjans' case, `blueriiot` - which apparently should be `hass-blueriiot`..

Looking at the code, the `TopicPrefix` _is_ `blueriiot` - why it's `sarah` and `hass-blueriiot` here - I don't know.

You could try changing this setting, for env vars its `HASS__TopicPrefix` and in the addon there should be a json setting under `HASS` and then `TopicPrefix`.

For @marcel-1984 - you should set yours to `sarah`, and @robertjan to `hass-blueriiot`. Then hopefully the discovery doc will sync up with the actual values being pushed, and you should receive data.

If either of you know _why_ these values would differ on your end, I'd be happy to know :P

**Comment by KikoSerranoH** [comment]
Hello, good afternoon.

I am new to all this and the truth is that I am quite lost.

I have installed ADDON and I have configured it as I show you:

![image](https://user-images.githubusercontent.com/91393764/233375814-27934c3d-bf2c-43e0-abaa-510b0a6c7b18.png)

But I only get the following sensors created:

![image](https://user-images.githubusercontent.com/91393764/233376093-21113b1d-e96e-4be2-a313-4707dafad1ea.png)

What do I still need to do to show me the sensors that show you?

It doesn't show me the temperature, chlorine, etc....

What am I doing wrong? The truth is that I am very lost in this subject and I do not know what to do.

For what it's worth I also show you the Log:

![image](https://user-images.githubusercontent.com/91393764/233376699-387fb7fe-ecaa-4f89-b6c7-9bf8e9d71957.png)


Thank you very much in advance.

Best regards.
